src/nvim/ui.coffee

```diff
@@ -73,15 +110,30 @@ class UI extends EventEmitter
     @cursor.style.height = @char_height + 'px'
 
     if config.blink_cursor
-      @cursor.classList.add('blink')
+      @start_cursor_blink()
     else
-      @cursor.classList.remove('blink')
+      @stop_cursor_blink()
+
+  start_cursor_blink: ->
+    @cursor.classList.add 'blink'
+
+  stop_cursor_blink: ->
+    @cursor.classList.remove 'blink'
 
   init_event_handlers: ->
+    resumeBlink = _.debounce (=> @start_cursor_blink()), 250
+    pauseBlink = =>
```

Comment:
Why not move `pauseBlink` outside and make it a member function.
Also we use `pause_blink` instead of `pauseBlink`, althought the latter is more "standard style".